### DISCUSSION TITLE
[General] <Decentralized Sequencer Structure>

### DISCUSSION THREAD

**Discussion by LHW0803** [zkSync-Community-Hub/zksync-developers#321] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

For a future decentralized sequencer, the rewards for the sequencer will be a native token? Or eth?
And to be a decentralized sequencer, what token will they have to stake?

### What is the specific issue or error you're encountering?

Question for the structures

### Can you share the error messages or logs you're receiving, if any?

Nope

### Have you made any recent changes to the contract before encountering this issue?

Nope

### Are there any external libraries or contracts that your contract interacts with?

-

### Can you provide the relevant portions of your contract code where the issue is occurring?

-

### Have you tried to isolate the problem, and if so, what were the results?

-

### What steps have you already taken to try to resolve the issue?

-

**Comment by MexicanAce** [comment]
Hi @LHW0803 👋 !

A decentralized sequencer is on our roadmap and we have no additional details at this moment.
